src/MyPlot/events/MyPlotPlayerEnterPlotEvent.php

```diff
@@ -9,19 +8,20 @@
 
 class MyPlotPlayerEnterPlotEvent extends MyPlotPlotEvent implements Cancellable{
 	public static $handlerList = null;
-	/** @var Player */
+	/** @var Player $player */
 	private $player;
 
 	/**
-	 * PlotEnterEvent constructor.
+	 * MyPlotPlayerEnterPlotEvent constructor.
+	 *
 	 * @param MyPlot $plugin
 	 * @param string $issuer
 	 * @param Plot $plot
 	 * @param Player $player
 	 */
 	public function __construct(MyPlot $plugin, string $issuer, Plot $plot, Player $player){
 		parent::__construct($plugin, $issuer, $plot);
-		$this->setPlayer($player);
```

Comment:
You DO know that setPlayer is a function in one of the parent classes, do you?

Reply:
The function is not in a parent class. Also, excess latency in the code is added when function calls are used instead of directly changing the variable, so his change is the result

Reply:
Ah yea.. always these 0.0000001µs, damn it.